test/qa/test_meshnet.py

```diff
@@ -25,10 +31,30 @@ def teardown_function(function):  # noqa: ARG001
 
 
 def test_meshnet_connect():
-    # Ideally peer update should happen through Notification Center, but that doesn't work often
-    sh.nordvpn.meshnet.peer.refresh()
     peer = meshnet.PeerList.from_str(sh.nordvpn.mesh.peer.list()).get_external_peer()
-    assert meshnet.is_peer_reachable(ssh_client, peer)
+    this_device = meshnet.PeerList.from_str(ssh_client.exec_command("nordvpn mesh peer list")).get_external_peer()
+
+    # Local machine -> Remote machine
+    nickname = "A-a-A"
```

Comment:
[minor] maybe set the nickname as `local-peer/machine` and `remote-peer/machine` instead, so then the comment is not needed anymore?